com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -9,11 +9,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - Added support for user-selected upscaling filters. Current options are automatic, bilinear, and nearest-neighbor.
 - Added batch mode support for the converters.
+- Soft shadow quality. Low, PCF 3x3 recommended for mobile. Medium, Tent 5x5. High, Tent 7x7.
```

Comment:
You can be a little bit more descriptive for users, such as, Added soft shadow quality option in the pipeline asset.

Reply:
Fixed.